src/App.tsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string | null;
```

Comment:
it's not clear to me what message , I am sure that this variable should be named differently
```suggestion
  pressedKey: string;
```